server/datastore/mysql/certificate_templates.go

```diff
@@ -239,7 +246,7 @@ SELECT
 FROM host_certificate_templates hct
 INNER JOIN certificate_templates ct ON hct.certificate_template_id = ct.id
 WHERE ct.team_id = ?
-GROUP BY 1`
+GROUP BY hct.status`
```

Comment:
There's another GROUP BY 1 below, maybe for clarity and consistency change that one too?